src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/FileStreamReadLightUp.cs

```diff
@@ -156,7 +156,7 @@ IntPtr overlapped
             );
 
             // older Windows systems:
-            [DllImport(@"kernel32.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
+            [DllImport(@"api-ms-win-core-file-l1-1-0.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
```

Comment:
@pallavit Please undo the change to this file and file a bug to follow up. If this is flagged by a Roslyn analyzer then a suppression pragma can be added for this file.